src/RequestForm.js

```diff
@@ -258,6 +265,37 @@ class RequestForm extends React.Component {
         this.setState({ blocked: true });
       }
     }
+
+    if (prevParentResources?.configs?.records[0]?.value !== parentResources?.configs?.records[0]?.value) {
+      const {
+        titleLevelRequestsFeatureEnabled: newTitleLevelRequestsFeatureEnabled,
+      } = this.getTlrSettings();
+
+      // eslint-disable-next-line react/no-did-update-set-state
+      this.setState(
+        {
+          titleLevelRequestsFeatureEnabled: newTitleLevelRequestsFeatureEnabled,
+        },
+        this.setTlrCheckboxInitialState(),
+      );
+    }
+  }
+
+  getTlrSettings() {
+    return getTlrSettings(this.props.parentResources?.configs?.records[0]?.value);
+  }
+
+  setTlrCheckboxInitialState() {
+    if (this.state.titleLevelRequestsFeatureEnabled === false) {
```

Comment:
this.state.titleLevelRequestsFeatureEnabled === false || this.props.query.itemId || this.props.query.itemBarcode

Reply:
won't be fixed as discussed